src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -88,10 +90,19 @@ public FeedbackSessionResponseStatus() {
         return noResponse;
     }
 
+    public List<String> getStudentsWhoRespondToAnyQuestion() {
+        Collections.sort(studentsResponded, compareByTeamNameStudentName);
+        return studentsResponded;
+    }
```

Comment:
A `get*` method causes a change in state by sorting an existing list? That doesn't sound right.